sdk/ml/azure-ai-ml/tests/command_job/e2etests/test_command_job.py

```diff
@@ -204,9 +205,12 @@ def test_command_job_builder(self, data_with_2_versions: str, client: MLClient)
         len(new_result.inputs) == 3
         assert result.display_name == new_result.display_name
 
-    @pytest.mark.skip(reason="Task 1791832: Inefficient, causing testing pipeline to time out.")
     @pytest.mark.timeout(900)
     @pytest.mark.e2etest
+    @pytest.mark.skipif(
+        condition=not sys.platform.startswith(("win32", "cygwin")),
+        reason="Skipping for PyPy as docker installation is not supported and skipped in dev_requirement.txt",
```

Comment:
Create a task to enable local test for non-win systems and update it in the reason.

Reply:
[Task 2287380](https://msdata.visualstudio.com/Vienna/_workitems/edit/2287380): [SDK - LocalJob] E2E test fails for Linux, Mac & PYPI